.gitmodules

```
@@ -0,0 +1,4 @@
+
+[submodule "documentation/_wiki"]
+	path = documentation/_wiki
+	url = https://github.com/openmsupply/application-manager-server.wiki.git
```

I found what was the problem with this:
```
Exporting docs to wiki repo...
[detached HEAD 69aa88b] Update documentation
 1 file changed, 7 insertions(+), 1 deletion(-)
fatal: You are not currently on a branch.
To push the history leading to the current (detached HEAD)
state now, use

    git push origin HEAD:<name-of-remote-branch>

error An unexpected error occurred: "Command failed.
```

```suggestion
	url = https://github.com/openmsupply/application-manager-server.wiki.git
	branch = master
```
We just need to specify what branch this submodule will initialise in. :)

---

Excellent, thanks 👍